lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -579,6 +583,33 @@ size_t _GenerateNetwork2TopoHash(const HdMaterialNetwork2& materialNetwork)
                     }
                 }
             }
+#else
+            // Hydra in USD 23.11 will add a "colorspace:Foo" parameter matching color managed "Foo"
+            // parameter:
+            auto isColorSpace = [](const TfToken& paramName) {
+                if (paramName.GetString().rfind(_tokens->ColorSpacePrefix.GetString(), 0) == 0) {
+                    return true;
+                }
+                return std::find(
+                           _mtlxKnownColorSpaceAttrs.begin(),
+                           _mtlxKnownColorSpaceAttrs.end(),
+                           paramName)
+                    != _mtlxKnownColorSpaceAttrs.end();
+            };
+
+            for (auto&& param : node.parameters) {
+                if (isColorSpace(param.first)) {
+                    MayaUsd::hash_combine(topoHash, hash_value(param.first));
```

Comment:
I see that there is a lot of overlap between this and lines 575-582. Is it worth sharing the code?

Reply:
The lambda can be refactored. The main loop has a different payload, so was left as-is.